pkgs/by-name/mi/mighty-mike/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  SDL2,
+  cmake,
+  makeWrapper,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation {
+  pname = "mighty-mike";
+  version = "3.0.2-unstable-2024-04-01";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = "MightyMike";
+    rev = "0a1d6c4c80a90ed6e333651cd0a438ec003cfbe5";
+    hash = "sha256-c7o0Q9KTbJhYOZ2c/V1EdV4ibdR3AnHTCZBManJQzrw=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+  ];
+  buildInputs = [ SDL2 ];
```

Comment:
```suggestion
  nativeBuildInputs = [
    SDL2
    cmake
    makeWrapper
  ];

  buildInputs = [ SDL2 ];

  strictDeps = true;
```

Reply:
@AndersonTorres Thanks! I've applied these changes and re-tested the build locally.